site/web/assets/css/librecores.css

```diff
@@ -519,3 +519,30 @@ h1 a:hover, h1 a:active {
 .bootstrap-select.btn-group .dropdown-menu {
   z-index: 1050; /* to over-ride navbar z-index*/
 }
+
+#delete-panel-button{
```

Comment:
Please add a space before the opening curly bracket, e.g. `#delete-panel-button: {` (for all CSS below)

Reply:
Done.